examples/nomad/ingress.hcl

```diff
@@ -28,4 +28,38 @@ nomad_ingress "nomad-http" {
   network  {
     name = "network.cloud"
   }
-}
\ No newline at end of file
+}
+
+nomad_ingress "fake-service-1" {
+  cluster  = "nomad_cluster.dev"
+  job = "example_1"
+  group = "fake_service"
+  task = "fake_service"
+
+  port {
+    local  = 19090
+    remote = "http"
+    host   = 19090
+  }
+
+  network  {
+    name = "network.cloud"
+  }
+}
+
+nomad_ingress "fake-service-2" {
+  cluster  = "nomad_cluster.dev"
+  job = "example_2"
+  group = "fake_service"
+  task = "fake_service"
+
+  port {
+    local  = 19090
+    remote = "http"
```

Comment:
can a number also be called?

Reply:
This is the label as defined in the Nomad stanza.

Ports in Nomad are always referenced by labels now, this value actually references the port name in Nomad. It could be a nice optimization to also look at the `static` block and allow a port number to be set this way but I worry it confuses rather than solves the problem.

```
group "app" {
  network {
    port "http" {
      to = 8080
    }

    port "9090" {
      to = 9090
    }
  }

  task "example" {
    driver = "docker"

    config {
      ports = ["http", "9090"]
    }
  }
}
```

https://www.nomadproject.io/docs/job-specification/network#port-parameters